### DISCUSSION TITLE
Ability to save and import settings for "Edit print parameters" tool

### DISCUSSION THREAD

**Discussion by ecospline** [sn4k3/UVtools#852] [Ideas]
Ability to save and import settings for "Edit print parameters" tool would greatly speed up changing the setup as it's quite repetable task - especially when you have to deal with lot of prototypes, where changes are made in model - not in print settings

**Comment by sn4k3** [comment]
Currently `UVtoolsCmd copy-parameters <input-file> <target-files>` allow you to copy parameters from file to files

**Comment by mjf55** [comment]
I was just going to ask for this feature when I found this searching archives for it.  Any chance of adding it to the GUI, as doing this in Widows stinks?

**Reply by sn4k3** [reply]
I can look at it.

**Comment by alpha8196** [comment]
hey I know this is an old post, but I was wandering the same thing, I know there's a save option now, but is it possible to load it?

**Reply by sn4k3** [reply]
Still not, but you can use UVtoolsCmd to set parameters in one line, if that helps...
